pysat/tests/test_instrument.py

```diff
@@ -1,21 +1,16 @@
 # -*- coding: utf-8 -*-
 # Test some of the basic _core functions
 import datetime as dt
+from importlib import reload as re_load
 import numpy as np
-import sys
 
-from nose.tools import raises
 import pandas as pds
+import pytest
 
 import pysat
 import pysat.instruments.pysat_testing
 import pysat.instruments.pysat_testing_xarray
 
-if sys.version_info[0] >= 3:
-    from importlib import reload as re_load
-else:
-    re_load = reload
-
```

Comment:
Do we need to maintain python 2.7 compliance in the unit tests?

Reply:
Only in the develop branch.

Reply:
OK.  Was debating whether to start pulling these out to clean up the code / reduce uncovered lines.

Reply:
For the develop-3 pulls, definitely the way to go.